lib/controllers/agendash.js

```diff
@@ -15,7 +15,11 @@ module.exports = function(agenda, options) {
         // Ignoring for now
       }
     });
-    agenda._mdb.admin().serverInfo((error, serverInfo) => {
+
+    // Mongoose internals changed at some point. This will fix crash for older versions.
+    const mdb = agenda._mdb.admin ? agenda._mdb : agenda._mdb.db;
```

Comment:
Not a biggie, but why not to just check for `agenda._mdb.db` instead of `.admin`?

Reply:
I was testing it locally for some time. This is how I got it eventually. I don't recall exactly why I wrote it this way. Sorry.

Reply:
No worries, thanks for working on it!